File: MonoProject.Model/VehicleOwnerDTO.cs
﻿using System;

namespace MonoProject.Models
{
    public class VehicleOwnerDTO
    {
        public Guid? Id { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public DateTime DateOfBirth { get; set; }
    }
}

File: MonoProject.React/src/common/VehicleOwnerDto.ts
export class VehicleOwnerDto {
    id: number | null;
    firstName: string;
    lastName: string;
    dateOfBirth: Date;
    constructor(id: number | null, firstName: string, lastName: string, dateOfBirth: Date,) {
        this.id = id;
        this.firstName = firstName;
        this.lastName = lastName;
        this.dateOfBirth = dateOfBirth;
    }
}